completion_aggregator/tasks.py

```diff
@@ -12,28 +13,32 @@
 from xblock.completable import XBlockCompletionMode
 from xblock.core import XBlock
 
-from django.conf import settings
-
+from . import compat
 from .models import Aggregator
 
 OLD_DATETIME = pytz.utc.localize(datetime(1900, 1, 1, 0, 0, 0))
 
+log = logging.getLogger(__name__)
+
 
 CompletionStats = namedtuple('CompletionStats', ['earned', 'possible', 'last_modified'])  # pylint: disable=invalid-name
 
 
 @shared_task
-def update_aggregators(user, course_key, block_keys=frozenset()):  # pylint: disable=unused-argument
+def update_aggregators(user, course_key, block_keys=(), force=False):  # pylint: disable=unused-argument
```

Comment:
Looks like all your arguments are used here, do you actually need the `#pylint: disable`?

Reply:
Nope.  And now I've enabled the useless-suppression lint, so this should be easier to spot in the future.